src/main/java/com/limechain/network/Network.java

```diff
@@ -43,6 +47,7 @@
 @Log
 public class Network {
     public static final String LOCAL_IPV4_TCP_ADDRESS = "/ip4/127.0.0.1/tcp/";
+    private final String peerIdKey = "nodePeerId";
```

Comment:
Rename it to just `peerId`. Not sure if this is the proper place for DB key constants... Maybe we should extract them into a separate class. What do you think.